runelite-client/src/main/java/net/runelite/client/plugins/npcunaggroarea/NpcAggroAreaPlugin.java

```diff
@@ -321,6 +331,28 @@ public void onNpcSpawned(NpcSpawned event)
 		checkAreaNpcs(event.getNpc());
 	}
 
+	@Subscribe
+	public void onNpcLootReceived(final NpcLootReceived npcLootReceived)
```

Comment:
This isn't the right event to use to check whether an NPC you killed died, since not all monsters drop loot, and even from monsters which do drop loot, there are situations where you would not be the recipient of it. You would be better off using a check similar to that used in the opponent info plugin.